docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,26 @@
+# Tizen IoT with Nubison Cloud
+
+The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an instrinsic capability of seamless connectivity.
```

Comment:
It looks like "instrinsic" should be fixed to "intrinsic".

Reply:
@bson1012  Again here, if you check my review comment, the spelling is correct but unfortunately while editing you have introduced a typo error. 
Kindly correct the spelling : 

"instrinsic" to "intrinsic"

Refer to :https://github.com/Samsung/tizen-docs/pull/560#discussion_r255876416

Reply:
@bson1012  can you please accommodate latest comments.

Reply:
done :)

Reply:
@Thanks @bson1012 for your quick response